### ISSUE TITLE
Windows, Service remove

### ISSUE THREAD

**Issue by wahmedswl** [takama/daemon#36]
Hi,
   Service remove should be silent, currently when service.Remove() is called, NSSM show dialogue to ask to continue.

```
To remove a service without confirmation:

        nssm remove <servicename> confirm
```

Thanks

**Comment by takama** [comment]
Ok, reasonable.
I have not any PC with Windows and I can not contribute in Windows part of the project. Sorry.
Could you make pull request for these fixes?

thank you

**Comment by 1for** [comment]
#48 here is the fixes.

**Comment by takama** [comment]
Closed due to #70
